testfiles/CoilWaterDesuperheatingUnitary.idf

```diff
@@ -0,0 +1,1639 @@
+!-Generator IDFEditor 1.34
```

Comment:
This file takes the original `CoileWaterDesuperheating` file and converts it to use the unitary system with a single-speed coil. We may not want to keep this extra file around, but this is the baseline I've been comparing against.